### PR TITLE
Script the docker version

### PR DESCRIPTION
**Pull request by alberto56**

### CODE REVIEW

**Comment by jpmckinney** [comment]
You're getting a `ModuleNotFoundError: No module named 'ca_on_candidates'` error in #222 because there is no such module :) Try ca_on_burlington instead.

**Comment by mirabuck** [comment]
Thanks for this--it will save folks on team a lot of headaches.

File: docker/README.md [hunk 1]
```diff
@@ -2,35 +2,19 @@
 
 ## Setup
 
-Build the environment:
+cd ./docker && ./deploy.sh
```
Comment (jpmckinney) [inline docker/README.md]: Please indent four spaces, so that it's rendered as a code block.

**Comment by alberto56** [comment]
Hi all, I have [made the proposed changes](https://github.com/opencivicdata/scrapers-ca/pull/225/commits/990e9afa760006a02935d996cda7daed168b0588) to the branch, and tested it locally, it works for me now!

**Comment by jpmckinney** [comment]
Thanks!

**Pull request merged**